src/node_util.cc

```diff
@@ -37,6 +39,21 @@ using v8::Value;
   VALUE_METHOD_MAP(V)
 #undef V
 
+static void GetProxyDetails(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  // Return undefined if it's not a proxy.
+  if (!args[0]->IsProxy())
+    return;
+
+  Local<Proxy> proxy = args[0].As<Proxy>();
+
+  Local<Array> ret = Array::New(env->isolate(), 2);
```

Comment:
If you use `args.GetIsolate()` here, you can drop the call to `Environment::GetCurrent()` a few lines up.